test/unit/containerDeploymentWebviewController.test.ts

```diff
@@ -311,6 +311,9 @@ suite("ContainerDeploymentWebviewController", () => {
 
     test("completeDockerStep reducer updates step status and handles success/failure", async () => {
         const addContainerConnectionStub = sinon.stub(controller as any, "addContainerConnection");
+        // Stub telemetry method
+        const sendActionEventStub = sinon.stub(telemetry, "sendActionEvent");
+        const sendErrorEventStub = sinon.stub(telemetry, "sendErrorEvent");

```

Comment:
There's a helper that you could use instead to consolidate behavior:
`const { sendErrorEvent } = stubTelemetry(sandbox);`